### PR TITLE
Add init command

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Comment by addyosmani** [comment]
Are you looking for feedback or is this mostly just to get the ball rolling before we iterate on init further in this PR? :)

**Comment by justinfagnani** [comment]
Right now I'm mostly saving my work, but comments are welcome. I should have this ready for real tomorrow.

**Comment by robdodson** [comment]
Any more thought to doing this with the existing generator? We could do a major rev PR?

**Comment by justinfagnani** [comment]
@robdodson I'm open to it, but I do think that we may want the workflow prescribed to settle first.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
     "command-line-commands": "^0.1.2",
     "polylint": "^2.10.0",
     "polyserve": "^0.7.1",
-    "web-component-tester": "^4.2.0"
+    "web-component-tester": "^4.2.0",
+    "yeoman-environment": "^1.5.2",
+    "yo": "^1.7.0"
```
Comment (garlicnation) [inline package.json]: Remove.

**Comment by garlicnation** [comment]
Is it possible to add a simple test for this?

**Comment by justinfagnani** [comment]
There's a simple test in the generator. Will add integration tests here
soon.

**Comment by garlicnation** [comment]
LGTM with package.json comment addressed.

**Pull request merged**